nightingale_dispatcher/src/nightingale_dispatcher/navigate_subtask.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+import rospy
+
+from nightingale_dispatcher.subtask import Subtask
+
+
+class NavigateSubtask(Subtask):
+    def __init__(self, name, room, feature, priority):
+        super.__init__(name, priority)
+
+        self.room = room
+        self.feature = feature
+
+        # TODO Consider making a Singleton or share resources
+        rospy.wait_for_service("/lookup_room_pose")
+        self.lookup_proxy = rospy.ServiceProxy("/lookup_room_pose", RoomLookup)
+
+    def execute(self):
```

Comment:
is this function complete? if not can u add todo or comment or raise unimplemented exception